src/components/AuthorViewpoint/FactListCard.tsx

```diff
@@ -0,0 +1,116 @@
+            (fact) => String(fact.id) == factId,
+        );
+        const newSelectedFacts = viewpointFactList.filter(
+            (fact) => String(fact.id) !== factId,
+        );
+        setViewpointFactList(newSelectedFacts);
+
+        //add the fact back to the searchData
+        const newSearchData = [...searchData, removedFact].filter(
+            (fact): fact is Fact => fact !== undefined,
+        );
+        setSearchData(newSearchData);
+
+        console.log(
+            "factlist",
+            newSelectedFacts.map((fact) => fact.id),
+        );
+    };
+
+    return (
+        <div className="h-full rounded-lg bg-neutral-100 px-7 py-4">
+            <h1 className="mb-1 text-lg font-semibold text-neutral-700">
+                事實
+            </h1>
+            <div className="mb-2 flex w-full items-center py-1 pr-[52px]">
+                <MagnifyingGlassIcon className="inline-block h-5 w-5 stroke-neutral-500" />
+                <Select
+                    variant="unstyled"
+                    searchable
+                    value={searchValue}
+                    onChange={(selectedFactId) => {
+                        setSelectedFactId(selectedFactId ? selectedFactId : "");
+                        console.log("User selected id:", selectedFactId);
+                    }}
+                    data={searchData.map((fact) => ({
+                        value: String(fact.id),
+                        label: fact.title,
+                    }))}
+                    checkIconPosition="right"
+                    radius={0}
+                    w="100%"
+                    limit={10}
+                    classNames={{
+                        input: "ml-2 bg-transparent text-lg font-normal text-neutral-500 focus-within:outline-b-2 focus-within:border-b-emerald-500 focus-within:outline-none",
+                    }}
+                    placeholder="搜尋 CommonGround"
+                    nothingFoundMessage="自行新增事實"
+                />
+            </div>
+            <div className="h-[calc(100vh-265px)] overflow-auto">
+                {/* 265px = 56px(header) + 69px(margin-top between header and this div) + 32px(padding-bottom of main)
+                + 92px(FactListCard title and search box) + 16px(FactListCard padding-bottom)*/}
+                <div className="flex flex-col gap-3 pl-7 pr-4">
+                    {viewpointFactList.map((fact) => (
+                        <EditViewpointFact
+                            key={fact.id}
+                            fact={fact}
+                            removeFact={removeFact}
+                        />
+                    ))}
+                </div>
+
+                <Button
+                    variant="transparent"
+                    leftSection={<PlusIcon className="h-6 w-6" />}
+                    classNames={{
+                        root: "px-0 ml-7 mt-2 text-neutral-600 text-base font-normal hover:text-emerald-500 duration-300",
```

Comment:
The current styling results in a padding whether there are facts before this or not. It makes less sense to add a padding here since the searchbar already has a y-padding. Consider moving the Button into the flexbox and utilizing the gap property.